uberduck_ml_dev/models/radtts.py

```diff
@@ -148,7 +135,8 @@ def __init__(
         )
         self.n_speaker_dim = n_speaker_dim
         assert self.n_speaker_dim % 2 == 0
-        self.speaker_embedding = torch.nn.Embedding(n_speakers, self.n_speaker_dim)
+        if n_speakers > 0:
```

Comment:
Let's be careful since this retroactively breaks code at load time while not changing performance.  Would rather put an assert in initialization.